### DISCUSSION TITLE
Changing blocks in the the default categories

### DISCUSSION THREAD

**Discussion by BlueDome77** [TurboWarp/extensions#540] [Ideas]
Any thoughts on adding or changing blocks in the default block categories similar to [PenguinMod](https://studio.penguinmod.site/editor.html)?

Maybe adding in advanced settings an option to show the default block pallet vs an updated block pallet?

![Screenshot 2023-06-07 at 7 05 54 AM](https://github.com/TurboWarp/extensions/assets/135789443/299005b0-9ac4-4ff8-9dd1-6968eed2e8e3)
![Screenshot 2023-06-07 at 7 06 26 AM](https://github.com/TurboWarp/extensions/assets/135789443/dabe6f14-6474-4bf1-9b37-2b9e411fa8b0)
![Screenshot 2023-06-07 at 7 06 33 AM](https://github.com/TurboWarp/extensions/assets/135789443/635f3f3e-1581-4669-8bb9-b0fcfea6b965)

**Comment by GarboMuffin** [comment]
Highly unlikely

**Comment by LilyMakesThings** [comment]
I have been working on an extension called TurboCharged that does this (image attached), however I wouldn't get your hopes up about it ever being on the gallery because there's a very good chance that it might not get accepted for being a little too unstable.
![image](https://github.com/TurboWarp/extensions/assets/127533508/79356387-ad58-4909-898f-c82ad34d6d2b)

And besides, it's not vanilla TurboWarp.

I think what makes TurboWarp far greater than PenguinMod is that it reserves new blocks to other extensions, which you can pick and choose what you want to be added to your block palette from. As opposed to PenguinMod which, for a while, just force-fed you operators you were never going to use.

**Reply by BlueDome77** [reply]
Would it have some of the blocks from PenguinMod?

I may have found the source code: https://github.com/PenguinMod/PenguinMod-Blocks/tree/develop/blocks_vertical

**Reply by LilyMakesThings** [reply]
I know where the pm source is, and obviously it would have some, but A: I'm not copying them directly from PM's source, and B: I'm not adding all of them

**Reply by BlueDome77** [reply]
Ok thanks!
